doc/tutorials/planning_around_objects/planning_around_objects.rst

```diff
@@ -31,10 +31,12 @@ First, update the target pose with the following change to make the robot plan t
     // Set a target Pose
     auto const target_pose = [] {
       geometry_msgs::msg::Pose msg;
-      msg.orientation.w = 1.0;
-      msg.position.x = 0.28;
-      msg.position.y = 0.4;  // <---- This value was changed
-      msg.position.z = 0.5;
+      msg.orientation.y = 0.8;
+      msg.orientation.w = 0.6;
+      msg.position.x = 0.1;
+      msg.position.y = 0.4;
+      msg.position.z = 0.4;
+      msg.orientation.w = 1.0; // All values are changed
```

Comment:
You have 2 lines setting `msg.orientation.w` -- I think there's only supposed to be 1?

Reply:
Oh yeah, my bad
I'll make the change

Reply:
Done!